internal/pkg/gitdirty/gitdirty.go

```diff
@@ -188,65 +212,78 @@ func remoteConvertHTTPStoSSH(httpsRemote string) (string, error) {
 // Based on regex, and may not match every possible style of remote, but tested on github and gitlab.
 //    Example input: git@git.test:testorg/testrepo.git
 //           output: https://git.test/testorg/testrepo.git
+//    Example input: git.test:testorg/testrepo.git
+//           output: https://git.test/testorg/testrepo.git
 func remoteConvertSSHtoHTTPS(sshRemote string) (string, error) {
 	if !isSSHRemote(sshRemote) {
 		return "", fmt.Errorf("%s is not an ssh remote", sshRemote)
 	}
 
-	httpsRemote := githubStyleSshRemoteRegexp.ReplaceAllString(sshRemote, "https://$1/$2")
+	var httpsRemote string
+	// Check if it has git@
+	if githubStyleSshRemoteRegexp.MatchString(sshRemote) {
+		httpsRemote = githubStyleSshRemoteRegexp.ReplaceAllString(sshRemote, "https://$1/$2")
+	} else {
+		// Doesn't have the git@ at the front of the url
+		httpsRemote = githubStyleSshRemoteNoAtRegexp.ReplaceAllString(sshRemote, "https://$1/$2")
+	}
+
 	if !isHTTPSRemote(httpsRemote) {
 		return "", fmt.Errorf("failed to convert ssh remote %q to https remote: got %q, which is not valid", sshRemote, httpsRemote)
 	}
 	return httpsRemote, nil
 }
 
-// nomralizeRemote returns a normalized form of the remote url.
-// The .git extension at the end of a remote url is optional for github
+// normalizeRemote returns a normalized form of the remote url.
+// The .git extension at the end of a remote url is optional for github.
+// Remote urls of type ssh may not start with git@, so this is trimmed.
 func normalizeRemote(remoteUrl string) string {
-	return strings.TrimRight(remoteUrl, ".git")
+	// Trim the git@ bc you can still have a remote url of type ssh w/o the git@
+	trimmedRemoteUrl := strings.TrimLeft(remoteUrl, "git@")
+	return strings.TrimRight(trimmedRemoteUrl, ".git")
 }
 
 // getRemoteName queries the repo at GitDirty.path for all remotes, and then
 // searches for the remote that matches the provided url, returning an error if
 // no remote url is found.
 // It will also attempt to match against different protocols - if an https protocol is
 // specified, if it can't find an exact match, it will look for an ssh-style match (and vice-versa)
-func getRemoteName(log hclog.Logger, repoPath string, remoteUrl string) (name string, err error) {
+func getRemoteName(log hclog.Logger, repoPath string, wpRemoteUrl string) (name string, err error) {
 	repo, err := git.PlainOpenWithOptions(repoPath, &git.PlainOpenOptions{
 		DetectDotGit: true,
 	})
 	if err != nil {
 		return "", errors.Wrapf(err, "failed to open git repo at path %q", repoPath)
 	}
 
-	remotes, err := repo.Remotes()
+	localRepoRemotes, err := repo.Remotes()
 	if err != nil {
-		return "", errors.Wrap(err, "failed to list remotes")
+		return "", errors.Wrap(err, "failed to list localRepoRemotes")
```

Comment:
I think we should keep the errors here to say things like "remote" and not the specific variable name. As a user/operator of Waypoint that receives logs or errors, they wont know what `localRepoRemotes` necessarily means but `remotes` is more intuitive as to where to look.

Reply:
Good catch. I agree and will update it accordingly.